npmrc

```diff
@@ -0,0 +1 @@
+registry=https://cli-npm.heroku.com
```

Comment:
We'd have to work with foundation to get an SSL cert setup and things, which is kind of a pain. Once we can see that this is working well we can change it.

In any case, `cli-npm.heroku.com` isn't going anywhere ever.